### ISSUE TITLE
RStudio products, including Cloud, and on-demand Server-pro and Connect on Azure

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/resources#165]
Today I met Pete Knast (<pete@rstudio.com>) from RStudio to share feedback about RStudio Cloud. RStudio Cloud helps us run the ds-incubator meetups, so it felt right to help them improve the produce. But the chat went beyond RStudio Cloud; we also discussed other products we might want to learn more about. Below copy all links.

--

> [RStudio Server Pro Standard for Azure](https://support.rstudio.com/hc/en-us/articles/360014725833-Getting-Started-with-RStudio-Server-Pro-Standard-for-Azure) is an on-demand, commercially-licensed integrated development environment (IDE) for the Microsoft Azure Cloud. It offers all of the capabilities found in the popular RStudio open source IDE plus turnkey convenience, enhanced security, the ability to manage multiple R versions and sessions, and more.

This could help someone like @2diiKlaus to get a powerful computing environment if and only whenever he needs it.

--

> [RStudio Team](https://rstudio.com/products/team/) ... includes RStudio Server Pro, RStudio Package Manager, and RStudio Connect. 

Someone like @jdhoffa, or @koopmand may want to know more about this products because banks may want to buy them -- particularly the Package Manager. Nichole Monhait (<nichole@rstudio.com>) from RStudio would be happy to show how this works.

--

> [RStudio Cloud Instructor](https://rstudio.cloud/plans/instructor) offers ... a deep discount for instructional use because ... for you to teach with RStudio Cloud. 

We can continue to use the free version but -- if we qualify -- this offer seems inexpensive  (~ $15/month).

**Comment by cjyetman** [comment]
Cool that you'll finally be able to use RStudio Cloud with a decent amount of RAM. That was an unfortunate deal breaker for something I was working on a year ago.

**Comment by 2diiKlaus** [comment]
Yes indeed, that is pretty cool and for sure an option for all data prep work we are doing with the large files. 
@Clare2D that might also be interesting for all historical work.

@maurolepore do you know how much this would roughly cost if we let's say use it for ~8 hrs/month, 32+ GB RAM

**Comment by cjyetman** [comment]
8GB RAM appears to be the limit, even for the paid accounts... but that's much better than the 1GB limit it has always had.

$34/month for non-profit orgs https://rstudio.cloud/plans/premium?discount=nonprofit

**Comment by Clare2D** [comment]
That does indeed sound interesting. Do we need this though? I guess for the funds work it would be most relevant? 
How does this work exactly? We open a project in the paid version of RStudio Cloud and it has more capacity to run a repository of our choice?

**Comment by maurolepore** [comment]
We [learned about RStudio Team](https://github.com/2DegreesInvesting/ds-incubator/issues/52). Some tools seem useful and many of them are free, or we could setup free alternatives. I may show people some of those tools in ds-incubator meetups. If we then love some of those tools we could subscribe for a free trial, and eventually consider buying what works best for us.
